invadedhouse/models/player.py

```diff
@@ -50,22 +52,3 @@ def in_inventory(self, item):
                 if myitem.get_name().lower() == item.lower():
                     return True
         return False
-
-
-class Inventory:
-
-    def __init__(self):
-        self.stuff = []
-
-    def add(self, item):
-        self.stuff.append(item)
-
-    def drop(self, item):
-        self.stuff.remove(item)
-        return item
-
-    def has(self, item):
-        return item in self.stuff
-
-    def all(self):
-        return self.stuff
```

Comment:
This all got moved to its own `inventory.py` file

Reply:
yep. cool